client/src/components/Login.jsx

```diff
@@ -1,20 +1,46 @@
 import React, { Component } from 'react';
 import RaisedButton from 'material-ui/RaisedButton';
 import TextField from 'material-ui/TextField';
+import { Link } from 'react-router';
+import $ from 'jquery';
+import { showErrorMessage } from '../utils/helperFunctions';
 
 class Login extends Component {
+
+  helperLogin() {
+    const user=$('#UNLogin').val();
+    const pass= $('#UNPass').val();
+    $.post("/login", { user: user, pass: pass }, (resp) => {
+      if (resp==="Succ") {
+        this.props.logIn(user);
+        this.context.router.push('/');
+      } else {
+        showErrorMessage("#LIMessages", 'Bad login', "badLogin");
+      }
+    });
+  }
+
   render() {
     return (
-      <div>
-        Username:<TextField /><br />
-        Password:<TextField type="password" /><br />
-        <RaisedButton label="Login" onClick={() => { this.props.change('SelectRoom')}} />
-        <RaisedButton label="Click to signup" onClick={() => { this.props.change('SelectRoom')}} />
+      <div id="loginContent">
+        Username:<TextField id="UNLogin" /><br />
+        Password:<TextField id="UNPass" type="password" /><br />
```

Comment:
Can you do `hintText="Hint Text"` and `floatingLabelText="Floating Label Text"` instead of sticking Password: in front